packages/askui-nodejs/src/core/model/custom-element-json.ts

```diff
@@ -1,16 +1,15 @@
 /**
  * Defines a 'custom element'. This is a UI element which is defined by
  * providing an image and other parameters such as degree of rotation.
- * It allows filtering for a UI element that is not recognized
- * by our machine learning models by default.
- * It can also be used for pixel assertions of elements using classical
- * [template matching](https://en.wikipedia.org/wiki/Template_matching).
+ * It allows filtering for a UI element based on an image instead of using
+ * text or element descriptions like `button().withText('Submit')` in
+ * `await aui.click().button().withText('Submit').exec()`.
  *
  * **Important:** The `CustomElementJson` needs to capture as accurately as possible
  * what the custom element looks like during test execution as otherwise
  * our machine learning models cannot find it, even with the additional data
  * provided. This is especially true for the resolution used while cropping
- * the `CustomElementJson.customImage` which should match the resolution during
+ * the `CustomElementJson.customImage` which should be close the resolution during
```

Comment:
the `CustomElementJson.customImage` which should be close to the resolution during

Reply:
What do you mean by that?

Reply:
I think that you meant the wording. I changed it a bit. See if you like it 🙂

Reply:
Yes, maybe it was just me but I thought the description was somehow vague 😄 . Great improvement on clarity (imho)!